cmd/tmsp-cli/tmsp-cli.go

```diff
@@ -15,10 +16,47 @@ import (
 	"github.com/urfave/cli"
 )
 
+//structure for data passed to print response
+// variables must be exposed for JSON to read
+type pr struct {
```

Comment:
Cool, thanks for the feedback/presentation! Any thoughts as to a better struct name? the current `pr` just comes from the fact it's fed to printResponse. Maybe `prVar`, `prOut`, or maybe something more to do with results, as it's essentially just a more detailed output of its variable `Res types.Results`. So with this in mind maybe... `expRes` or `expRslt` for expanded results